asciidoctorj-arquillian-extension/build.gradle

```diff
@@ -0,0 +1,28 @@
+dependencies {
+  compile project(path: ':asciidoctorj')
+  compile project(path: ':asciidoctorj-test-support', configuration: 'tests')
+  compile "org.jboss.arquillian.container:arquillian-container-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-spi:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-impl-base:$arquillianVersion"
+  compile "org.jboss.arquillian.container:arquillian-container-test-impl-base:$arquillianVersion"
+}
+
+jar.dependsOn project(':asciidoctorj').testClasses
+
+jar {
+  from project(':asciidoctorj').sourceSets.test.output
+  include 'org/asciidoctor/arquillian/**/*'
```

Comment:
Thanks for the clarification. Splitting it is maybe a good idea.
I tried to use the extension in an example project. 

```
apply plugin: 'groovy'
apply plugin: 'idea'

group 'de.calmdevelopment'
version '0.1'

defaultTasks 'clean', 'build', 'eclipse'

repositories {
  mavenLocal()
  jcenter()
}

dependencies {
  compile 'org.codehaus.groovy:groovy-all:2.4.3'
  testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
  testCompile 'org.asciidoctor:asciidoctorj-arquillian-extension:1.5.3-SNAPSHOT'
}
```

Just to see the dependency-graph:

```
testCompile - Compile classpath for source set 'test'.
+--- org.codehaus.groovy:groovy-all:2.4.3
+--- org.spockframework:spock-core:1.0-groovy-2.4
|    +--- org.codehaus.groovy:groovy-all:2.4.1 -> 2.4.3
|    \--- junit:junit:4.12
|         \--- org.hamcrest:hamcrest-core:1.3
\--- org.asciidoctor:asciidoctorj-arquillian-extension:1.5.3-SNAPSHOT
     +--- org.asciidoctor:asciidoctorj:1.5.3-SNAPSHOT
     |    +--- org.jruby:jruby-complete:1.7.20
     |    \--- com.beust:jcommander:1.35
     +--- org.asciidoctor:asciidoctorj-test-support:1.5.3-SNAPSHOT
     |    \--- junit:junit:4.12 (*)
     +--- org.jboss.arquillian.container:arquillian-container-spi:1.1.8.Final
     |    +--- org.jboss.arquillian.core:arquillian-core-spi:1.1.8.Final
     |    |    \--- org.jboss.arquillian.core:arquillian-core-api:1.1.8.Final
     |    +--- org.jboss.arquillian.config:arquillian-config-api:1.1.8.Final
     |    |    \--- org.jboss.shrinkwrap.descriptors:shrinkwrap-descriptors-api-base:2.0.0-alpha-7
     |    +--- org.jboss.arquillian.config:arquillian-config-impl-base:1.1.8.Final
     |    |    +--- org.jboss.arquillian.config:arquillian-config-api:1.1.8.Final (*)
     |    |    +--- org.jboss.arquillian.core:arquillian-core-api:1.1.8.Final
     |    |    +--- org.jboss.arquillian.core:arquillian-core-spi:1.1.8.Final (*)
     |    |    \--- org.jboss.shrinkwrap.descriptors:shrinkwrap-descriptors-spi:2.0.0-alpha-7
     |    |         \--- org.jboss.shrinkwrap.descriptors:shrinkwrap-descriptors-api-base:2.0.0-alpha-7
     |    +--- org.jboss.shrinkwrap:shrinkwrap-api:1.2.2
     |    \--- org.jboss.shrinkwrap.descriptors:shrinkwrap-descriptors-api-base:2.0.0-alpha-7
     +--- org.jboss.arquillian.container:arquillian-container-test-spi:1.1.8.Final
     |    +--- org.jboss.arquillian.container:arquillian-container-spi:1.1.8.Final (*)
     |    +--- org.jboss.arquillian.test:arquillian-test-spi:1.1.8.Final
     |    |    +--- org.jboss.arquillian.core:arquillian-core-spi:1.1.8.Final (*)
     |    |    \--- org.jboss.arquillian.test:arquillian-test-api:1.1.8.Final
     |    |         \--- org.jboss.arquillian.core:arquillian-core-api:1.1.8.Final
     |    \--- org.jboss.arquillian.container:arquillian-container-test-api:1.1.8.Final
     |         \--- org.jboss.shrinkwrap:shrinkwrap-api:1.2.2
     +--- org.jboss.arquillian.container:arquillian-container-impl-base:1.1.8.Final
     |    +--- org.jboss.arquillian.core:arquillian-core-api:1.1.8.Final
     |    +--- org.jboss.arquillian.core:arquillian-core-spi:1.1.8.Final (*)
     |    +--- org.jboss.arquillian.config:arquillian-config-api:1.1.8.Final (*)
     |    +--- org.jboss.arquillian.config:arquillian-config-impl-base:1.1.8.Final (*)
     |    +--- org.jboss.arquillian.container:arquillian-container-spi:1.1.8.Final (*)
     |    \--- org.jboss.shrinkwrap.descriptors:shrinkwrap-descriptors-spi:2.0.0-alpha-7 (*)
     \--- org.jboss.arquillian.container:arquillian-container-test-impl-base:1.1.8.Final
          +--- org.jboss.arquillian.container:arquillian-container-spi:1.1.8.Final (*)
          +--- org.jboss.arquillian.test:arquillian-test-api:1.1.8.Final (*)
          +--- org.jboss.arquillian.container:arquillian-container-test-api:1.1.8.Final (*)
          \--- org.jboss.arquillian.container:arquillian-container-test-spi:1.1.8.Final (*)
...
```

asciidoctorj as full dependency seems a bit to much.